app/services/ect_at_school_periods/switch_mentor.rb

```diff
@@ -43,7 +43,8 @@ def create_training_period!
         started_on: earliest_possible_start_date,
         school_partnership: earliest_matching_school_partnership,
         expression_of_interest:,
-        author:
+        author:,
+        mentee: ect_at_school_period
```

Comment:
`Schedules::Find` only flags replacements when it receives a mentee but we still create mentor training periods elsewhere without it, e.g: 

https://github.com/DFE-Digital/register-early-career-teachers-public/blob/main/app/services/schools/register_mentor.rb#L66-L70

Should these flows also pass the mentee in so we don't default to standard schedules there?